Medusa can't use tsconfig.json path aliases

After updating my `tsconfig.json` to use path aliases, the app won't run. It stars to throwing errors: `Error in loading config: Cannot find module '@app/modules/whatever'`.

## Code Snippets
```typescript
// tsconfig.json
// ...
"baseUrl": ".",
"paths": {
  "@app/*": ["src/*"],
},
// ...
```

--- a/packages/cli/medusa-cli/cli.js
+++ b/packages/cli/medusa-cli/cli.js
@@ -2,6 +2,7 @@
 
 try {
   require("ts-node").register({})
+  require("tsconfig-paths").register({})
 } catch {}
 require("dotenv").config()
 require("./dist/index.js")

